### ISSUE TITLE
drag&drop text files

### ISSUE THREAD

**Issue by 7flash** [zsviczian/obsidian-excalidraw-plugin#781]
**Is your feature request related to a problem? Please describe.**
It often happens I do manually copy-paste content of text files into excalidraw document

**Describe the solution you'd like**
It would be convenient to be able drag&drop file directly into the document and see it's content emplaces as text element

**Comment by zsviczian** [comment]
You mean a .txt file from your system?

**Comment by 7flash** [comment]
Yes, when I drag&drop them it does emplace the path to file, but I wish that would also actually emplace content of the file. Perhaps can be implemented to fetch from file system similarly how it's done in my script to fetch remote files: https://github.com/zsviczian/obsidian-excalidraw-plugin/pull/780
